### PR TITLE
#143358919 Integrate HoundCI

### PR DESCRIPTION
**Pull request by andela-onnenanya**
#### What does this PR do?
Integrate HoundCI with badge attached to readme
#### Description of Task to be completed?
Integrate HoundCI with badge attached to readme. This is to automatically validate the codebase with standard style guides
#### How should this be manually tested?
Push to github and make a PR
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Integrate HoundCI with badge attached to readme
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```
Comment (emmyakin) [inline .hound.yml]: change this to the eslint config filename .eslintrc

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+  ignore_file: .eslintignore
```
Comment (emmyakin) [inline .hound.yml]: do we have .eslintignore file?

**Review by emmyakin — CHANGES_REQUESTED** [review]

File: .hound.yml (hunk 1 above)
Comment (andela-onnenanya) [reply .hound.yml]: Ok, on it now

File: .hound.yml (hunk 2 above)
Comment (andela-onnenanya) [reply .hound.yml]: No, should I remove it?

**Review by emmyakin — APPROVED** [review]
Thanks

**Pull request merged**